### DISCUSSION TITLE
Music

### DISCUSSION THREAD

**Discussion by RuckusMcGuckus** [bucanero/pkgi-ps3#138] [Q&A] [answered]
Loving PKGI. First time modding my ps3 and this community makes it very easy to do so. Thank you for this awesome mod

Just wondering what is the theme music for pkgi? I Can't find it using reverse search apps. I kinda like it, sounds like some rpg music

**Comment by bucanero** [comment, marked as answer]
> Just wondering what is the theme music for pkgi? I Can't find it using reverse search apps. I kinda like it, sounds like some rpg music

the background music is a S3M tracker file by [Nobody/Wild Light](https://github.com/nobodo)

You can download the original file here: https://github.com/bucanero/apollo-ps3/blob/master/data/haiku.s3m

**Reply by RuckusMcGuckus** [reply]
That was quick, thanks
